### PR TITLE
Cell-Loaded Weapons reworks and implementations

### PR DESCRIPTION
**Pull request by Heroman3003**
tl;dr: Finishes work on ML-3M, adds to R&D, buffs HoS's NSFW and adds prototype NSFW, adds proper hybrid of ML-3M and NSFW for ERT.

In more details:

NSFW:
- Adds prototype cell-loaded revolver: basically NSFW, but can only hold prototype clip
- Adds prototype combat clip: can hold all cells regular nsfw clip can, but only has two slots
- HoS' NSFW clip expanded to hold four cells
- HoS' NSFW kit now spawns with extra STUN cell
- All NSFW cells (except FINAL OPTION), prototype NSFW and prototype clip can be made in R&D

ML-3M:
- Changed displayed name to cell-loaded medigun/advanced cell-loaded medigun (official name still displayed in the description)
- Changed gun balancing: advanced version no longer is any different from regular beyond extra visual appeal
- Both versions can hold both regular medical cell clip and advanced one
- Regular clip holds 3 cells. Advanced clip holds 6 cells
- Regular ML-3M kit spawns with regular medigun, a regular clip, and cells: BRUTE, BURN and STABILIZE
- Advanced ML-3M kit spawns with advanced medigun, a regular clip and cells: BRUTE, BURN, STABILIZE, TOXIN and OMNI
- Heal numbers from the cells reduced across the board
- Reduced buff numbers from HASTE and RESIST cells
- Added extra tier of upgrade: BRUTE-III, BURN-III, TOXIN-III and OMNI-III, to server as anoma-tech level cells
- Regular medigun, both clips and all cells, existing and new, can be made in R&D
- Added a regular ML-3M kit to medical lower storage
- Added an advanced ML-3M kit to CMO locker
- Removed regular ML-3M kit from random medical item pool

Hybrid:
- A hybrid already existed, but to re-describe: amalgamation that can use any clip, combat, medical, or hybrid clip
- Hybrid clips: can hold cells both medical and combat, or intermixed. Come in versions with 3 and with 6 cell slots.
- Neither hybrid gun nor hybrid clips are obtainable outside adminspawn
- A regular hybrid kit has: hybrid gun, advanced hybrid clip and following cells: combat: 3 STUN, 2 NET, and medical: BRUTE-III, BURN-III, TOXIN-III, OMNI-III, STABILIZE-II
- A military hybrid kit has: hybrid gun, advanced hybrid clip and following cells: combat: 2 SCATTERSTUN, 3 LETHAL, ION, XRAY, and medical: STABILIZE-II, HASTE and RESIST
- A single regular hybrid kit replaces NSFW and pre-mapped cells in ERT armory
- Military hybrid kits replace generic combat NSFW kits in HAP armory

### CODE REVIEW

File: maps/tether/tether-10-colony.dmm [hunk 1]
```diff
@@ -1158,16 +1158,7 @@
...
-/obj/item/ammo_casing/nsfw_batt/combat/stun,
-/obj/item/ammo_casing/nsfw_batt/combat/stun,
-/obj/item/ammo_casing/nsfw_batt/medical/brute2,
-/obj/item/ammo_casing/nsfw_batt/medical/burn2,
-/obj/item/ammo_casing/nsfw_batt/medical/stabilize2,
-/obj/item/ammo_casing/nsfw_batt/combat/net,
-/obj/item/ammo_casing/nsfw_batt/combat/net,
-/obj/item/ammo_magazine/nsfw_mag,
-/obj/item/weapon/gun/projectile/nsfw,
+/obj/item/weapon/storage/secure/briefcase/nsfw_pack_hybrid,
```
Comment (Novacat) [inline maps/tether/tether-10-colony.dmm]: Make sure to replace the HAP NSFW kits with hybrid ones.

Comment (Heroman3003) [reply maps/tether/tether-10-colony.dmm]: Added a special kit just for HAP armory instead
